lib/stripe/stripe_client.rb

```diff
@@ -99,6 +109,11 @@ def request
 
     def execute_request(method, path,
                         api_base: nil, api_key: nil, headers: {}, params: {})
+      raise ArgumentError, "method should be a symbol" \
+        unless method.is_a?(Symbol)
```

Comment:
We have this check in `ConnectManager#execute_request` anyway. I hoisted it up here just so that it's more clear what the shape of this variable is when reading `StripeClient`.